src/search.jl

```diff
@@ -62,6 +62,64 @@ function spectrum(ψ::MPS, keep::Int)
     states[:, 1:keep], prob[1:keep], pCut
 end
 
+# ρ needs to be in the right canonical form
+function spectrum_new(ψ::MPS, keep::Int) 
+    @assert keep > 0 "Number of states has to be > 0"
+    T = eltype(ψ)
+    
+    keep_extra = keep
+    lpCut = -1000
+    k = 1
+
+    if keep < (*)(rank(ψ)...)
+        keep_extra += 1
+    end
+    lprob=zeros(T,k)
```

Comment:
Please fix code styling:
1. `=` should have a space before and after it
2. after commas, there should be a space